### ISSUE TITLE
many `Cannot read property 'x' of undefined` & `Cannot read property 'lastChild' of null` errors

### ISSUE THREAD

**Issue by misterburton** [dimsemenov/PhotoSwipe#1096]
Large (56 photos), list of links style gallery. The errors begin at image 22 & fire on many ensuing images. The gallery is still somewhat navigable on desktop after the errors fire & wholly broken on mobile:

http://photoshopbattles.ideo.com/

I've tried turning off the history API w/ no luck. The only .js i'm using for PhotoSwipe is the pure Vanilla JS implementation from the `getting-started` page. Any help is greatly appreciated.

**Comment by stephentuso** [comment]
I was having a similar problem, looks like this can happen if the values for `w` and `h` are invalid. I was just passing on the width and height properties from the file data returned from the server, I didn't look at it close enough and it turned out those properties were arrays. When it was an array of only one item, it seemed that photoswipe would pull out the number, but occasionally the arrays have two items and this error would be thrown. I'm passing the numbers correctly now and the error is gone.

tldr: Make sure that `w` and `h` are always valid numbers
